core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -171,6 +178,46 @@ var TopicEditorPage = function() {
     confirmSubtopicCreationButton.click();
   };
 
+  this.dragSkillToSubtopic = function(skillIndex, subtopicIndex) {
+    var target = subtopicTitles.get(subtopicIndex);
+    var toMove = skillCards.get(skillIndex);
+    browser.executeScript(
+      dragAndDrop, toMove.getWebElement(), target.getWebElement());
+  };
+
+  this.dragSkillBetweenSubtopics = function(
+      fromSubtopicIndex, skillCardIndex, toSubtopicIndex) {
+    var subtopicCol = subtopicColumns.get(fromSubtopicIndex);
+    skillNamesElems = subtopicCol.all(
+      by.css('.protractor-test-assigned-skill-card-text'));
+    var toMove = skillNamesElems.get(skillCardIndex);
+    var target = subtopicColumns.get(toSubtopicIndex);
+    browser.executeScript(
```

Comment:
Hey @U8NWXD This PR is great. Just one small thing, should be add this ```browser.exectueScript``` to a seperate function, something like ```dragAndDrop(var fromElement, var toElement)``` since this is being used in two places and maybe be used at more places in the future. Also it will improve the code readibility.
@nithusha21

Reply:
Done in 4878691de